mmv1/api/resource/docs.go

```diff
@@ -32,5 +32,7 @@ type Docs struct {
 
 	OptionalProperties string `yaml:"optional_properties"`
 
+	WriteOnlyProperties string `yaml:"write_only_properties"`
```

Comment:
Please give yourself a followup to add guidance on _adding_ WO fields to the contributor guide.